content/developer/reference/backend/data.rst

```diff
@@ -268,7 +268,8 @@ creating a number of simple records of the same model in bulk.
 For this case, data files can also use csv_, this is often the case for
 :ref:`access rights <reference/security/acl>`:
 
-* the file name is :file:`{model_name}.csv`
+* the file name is :file:`{model.name}.csv` (use the exact content of
+  the variable ``_name``)
```

Comment:
You can add a reference to the attribute, something like that: 
```suggestion
* the file name is :file:`{model_name}.csv` (model_name
should be :attr:`~odoo.models.BaseModel._name`)
```

Reply:
here since we're in the reference doc, we're not referring to a specific model. Is the :attr: going to work as intended and let the reader understand that generically, you should name your CSV according to the targeted model name (whatever that model is)? Otherwise I can make the reference to the exact attribute in an example instead. Wdyt?